test/Microsoft.DocAsCode.Build.Engine.Tests/DocumentBuilderTest.cs

```diff
@@ -796,9 +796,8 @@ public void TestBuildWithInvalidPathWithTokenAndMapping()
                         "\n",
                         "<p><a href=\"invalid-a.md\">link a</a>",
                         "<a href=\"../b/invalid-b.md\">link b</a></p>",
-                        $"<!-- BEGIN INCLUDE: Include content from &quot;{_inputFolder}/b/token.md&quot; --><p><a href=\"invalid-a.md\">link a</a>",
-                        "<a href=\"../b/invalid-b.md\">link b</a></p>",
-                        "<!--END INCLUDE -->"),
+                        $"<p><a href=\"invalid-a.md\">link a</a>",
+                        "<a href=\"../b/invalid-b.md\">link b</a></p>"),
```

Comment:
Ach, sorry... I fixed a ton of tests, but I can't actually run them locally for some reason, whenever I try I get an error 

CS1705	Assembly 'Microsoft.DocAsCode.Build.Engine' with identity 'Microsoft.DocAsCode.Build.Engine, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null' uses 'jint, Version=2.10.3.0, Culture=neutral, PublicKeyToken=2e92ba9c8d81157f' which has a higher version than referenced assembly 'Jint' with identity 'Jint, Version=2.5.0.0, Culture=neutral, PublicKeyToken=2e92ba9c8d81157f'	Microsoft.DocAsCode.Build.Engine.Tests..NET Framework 4.5.2	C:\Repos\CSI\docfx\test\Microsoft.DocAsCode.Build.Engine.Tests\ConvertStrongTypeToJsValueTest.cs	22	Active